File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.py
N = int(input())
res = N
i = 0

while True:
    if res<10:
        res*10
    A = res // 10
    B = res % 10
    C = A+B
    res = (B*10) + C%10
    i+=1
    if res == N :
        break
print(i)

File: 백준/Bronze/1110. 더하기 사이클/더하기 사이클.js
var fs = require('fs');
var input = Number(fs.readFileSync('/dev/stdin').toString().trim());

var num = input;
var t;
var cnt = 0;

while (true){
    cnt ++;

    t = parseInt(num/10) + num % 10;
    num = (num % 10) * 10 + t % 10; 

    if (num === input){
        break;
    }
}

console.log(cnt);